README.md

```diff
@@ -55,122 +91,67 @@ var i18n = require('hmpo-i18n');
+    req.setLanguage(req.query.lang);
+    
     // a translate method is now available on the request
     // this will translate keys according to the language request headers
     res.render('index', {
-        title: req.translate('title')
+        title: req.translate('title')        
+        message: req.translate('greeting');
     });
 });
 
 app.listen(3000);
+...
 ```
+## Configuration Options
+
+| Option              | Description |
+|---------------------|-------------|
+| `baseDir`           | Root directory for resource lookup based on path. Default: process.cwd(). 
+| `allowedLangs`      | List of allowed languages 
+| `locale`            | Default locale
+| `cookie`            | Cookie settings (e.g., `{ name: 'lang' }`) 
+| `query`             | Name of query parameter 
+| `noCache`           | Disable caching of resources 
+| `watch`             | Watch for resource file changes 
+| `setHeaders`        | Set Content-Language header 
+| `fallbackLang`      | Sets the fallback language if none is specified or a key is missing. Default: 'en'.
+| `fallbackNamespace` | Sets the fallback namespace if unspecified or key is missing. Default: 'default'.
+| `path`              | For the fs loader, sets the resource file path and pattern. Default: locales/__lng__/__ns__.__ext__.
+| `backend`           | Sets a custom backend (must have load method) for non-fs loading. Default: [fs resource loader](./lib/backends/fs.js)
 
-### As express middleware with custom language detection:
+### Base directory
 
-Middleware can detect the language from the `Accepts` header by if the `detect` option is true.
-A custom language can be set using `req.setLanguage(lang)`.
+The root directory in which to look for resources on the path defined - Default: process.cwd()
 
 ```javascript
-var app = require('express')();
-
-var i18n = require('hmpo-i18n');
-
-i18n.middleware(app);
-
-app.use(function (req, res, next) {
-    // load language from querystring parameter - e.g. ?lang=en
-    req.setLanguage(req.query.lang);
-    next();
+i18n({
+    baseDir: __dirname
 });
-...
 ```
 
-### Passing language and namespace options
+Note: if you are using hmpo-i18n inside a module which is likely to be installed as a child dependency of another project then it is highly recommended that you set this property.
 
-The translate method takes an additional options object with parameters for language and namespace to be loaded. This applies to both standalone and express middleware options (although in the middleware case any language option will overwrite the accepts header as a language preference)
+### Single and Multi-Language Support Example
 
-If the options are set to an array or a string, then this will be used as the language.
+If multiple langauges are passed, then the first matching language will be used. For example: i18n.translate('greeting', ['en-GB', 'en']); will look for en-GB first, and fall back to en only if a translation for en-GB is not found.
```

Comment:
I would still mention in the text here you can pass in a string or an array as the example shows